### ISSUE TITLE
false positive on braced expressions

### ISSUE THREAD

**Issue by graingert** [flake8-commas/flake8-commas#12]
``` python
x = (
    a +
    b +
    c
)
```

becomes:

``` python
x = (
    a +
    b +
    c,
)
```
